### DISCUSSION TITLE
CacheInputFileSystem default cache time

### DISCUSSION THREAD

**Discussion by markjm** [webpack/webpack#15147] [Ideas]
Basically, similar question as #15036. `CacheInputFileSystem` only caches for 1 minute. In longer builds, useful fs entries will be purged away, requiring a second identical fs call. One example that stands out is https://github.com/xz64/license-webpack-plugin, which does a bunch of fs calls very late in the compilation process to grab license information.

It seems 1min was chosen arbitrarily - what are y'alls thoughts on (1) raising it much higher, or (2) perhaps allowing setting the value via config w/ default

https://github.com/webpack/webpack/blob/fbcc470bcf6a2fcf7494f6f720638d06a5166ea2/lib/node/NodeEnvironmentPlugin.js#L44

cc @alexander-akait

**Comment by alexander-akait** [comment]
@sokra Why we have `60000`? Maybe something what we don't know?

**Reply by markjm** [reply]
As further note, when running in any sort of long-running/watch mode, it seems we do properly `purge()` old entries - not to say we cant possibly return old cached data, but seems we could do that within 1min as well
